### PR TITLE
updated just models

### PR DESCRIPTION
**Pull request by satra**

### CODE REVIEW

**Comment by yarikoptic** [comment]
Marked as "internal" since AFAIK those are not yet affecting the client's functionality, merging

File: dandi/models.py [hunk 1]
```diff
@@ -179,6 +181,15 @@ class SpeciesType(TypeModel):
...
 
+    dxdate: Optional[List[date]] = Field(
+        None,
+        title="Dates of diagnosis",
+        description="Dates of diagnosis",
+        readOnly=True,
+        nskey="dandi",
+        rangeIncludes="schema:Date",
+    )
+
```
Comment (yarikoptic) [inline dandi/models.py:192]: hm, shouldn't this (and possibly other changes) have affected the produced json schema?  according to the workflow no changes being pushed to `dandi/schema` repository by https://github.com/dandi/dandi-cli/runs/1476406805?check_suite_focus=true

Comment (satra) [reply dandi/models.py:192]: it should have.

Comment (yarikoptic) [reply dandi/models.py:192]: filed #289 289

**Pull request merged**